### PR TITLE
Add security file

### PR DESCRIPTION
**Pull request by Letme**
For some reason @JasperCraeghs removed this file, but it is needed for the Best Practices badge. I have also added it to documentation, so that it is not considered obsoleted.

### CODE REVIEW

File: docs/security.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.. include:: ../SECURITY.md
```
Comment (JasperCraeghs) [inline docs/security.rst:1]: `warnings-plugin/docs/index.rst:5: WARNING: toctree contains reference to document 'security' that doesn't have a title: no link will be generated`

Sphinx can't handle Markdown, but GitHub supports RST.

**Review by JasperCraeghs — CHANGES_REQUESTED** [review]

File: docs/security.rst (hunk 1 above)
Comment (Letme) [reply docs/security.rst:1]: Indeed: efac4e5

Comment (JasperCraeghs) [reply docs/security.rst:1]: Is there a way to align the file extension and still include it here?

Comment (Letme) [reply docs/security.rst:1]: Contribution guide is included in same way. I think its now faiking for some changes in Sphinx conf.py

**Review by JasperCraeghs — APPROVED** [review]

**Pull request merged**